warehouse/query-core/src/main/java/datawave/query/tables/ShardQueryLogic.java

```diff
@@ -167,7 +167,7 @@
     protected ScannerFactory scannerFactory = null;
     protected Scheduler scheduler = null;
     protected EventQueryDataDecoratorTransformer eventQueryDataDecoratorTransformer = null;
-    protected ShardQueryConfiguration config = ShardQueryConfiguration.create();
+    protected ShardQueryConfiguration config;
```

Comment:
should we make this private like we did for baseConfig?

Reply:
sure